c7n/reports/csvout.py

```diff
@@ -96,7 +96,7 @@ def report(policies, start_date, options, output_fh, raw_output_fh=None):
 
         records += policy_records
 
-    rows = formatter.to_csv(records)
+    rows = formatter.to_csv(records, not options.all_findings)
```

Comment:
Since `to_csv` has optional keyword args for both `reverse` and `unique`, I think we'll want to explicitly use `unique=` here.